sensor/kubernetes/listener/resources/rbac/binding_fetcher.go

```diff
@@ -7,16 +7,26 @@ import (
 	"github.com/stackrox/rox/generated/internalapi/central"
 	"github.com/stackrox/rox/pkg/errorhelpers"
 	pkgKubernetes "github.com/stackrox/rox/pkg/kubernetes"
+	"github.com/stackrox/rox/pkg/retry"
```

Comment:
I was not aware we have this package I thought about using `"k8s.io/client-go/util/retry"`

Reply:
On the other hand maybe we should create k8s client with auto-retry like we have it here for jira https://github.com/stackrox/junit2jira/pull/35
but this will affect all components

Reply:
That's something that we can explore but I would keep it out of the scope of this PR because the k8s client is used in many components and we need to make sure we do not break anything.